providers/edge3/src/airflow/providers/edge3/cli/edge_command.py

```diff
@@ -517,12 +517,18 @@ def _launch_worker(args):
 @providers_configuration_loaded
 def worker(args):
     """Start Airflow Edge Worker."""
+    if args.umask:
+        umask = args.umask
+    else:
+        umask = conf.get("edge", "worker_umask", fallback=settings.DAEMON_UMASK)
```

Comment:
As you create a new setting, can you also please add this to `providers/edge3/provider.yaml` including some descriptions?

By the way in Python this if/else can be made shorter as:
```suggestion
    umask = args.umask or conf.get("edge", "worker_umask", fallback=settings.DAEMON_UMASK)
```